src/sidebar/test/annotation-metadata-test.js

```diff
@@ -247,6 +249,26 @@ describe('annotation-metadata', function () {
     });
   });
 
+  describe('.isPublic', function () {
+    it('returns true if an annotation is shared within a group', function () {
+      assert.isTrue(annotationMetadata.isPublic(fixtures.publicAnnotation()));
+    });
+
+    unroll('returns false if an annotation is not publicly readable', function (testCase) {
+      var annotation = Object.assign(fixtures.defaultAnnotation(), {permissions: testCase});
+      assert.isFalse(annotationMetadata.isPublic(annotation));
+    }, [{
+      read:['acct:someemail@localhost'],
+    }, {
+      read:['something invalid'],
+    }]);
+
+    it('returns false if an annotation is missing permissions', function () {
+      var annotation = Object.assign(fixtures.defaultAnnotation());
```

Comment:
The `Object.assign()` here has no effect because it only has one argument. Perhaps you meant `Object.assign(fixtures.defaultAnnotation(), {permissions: null})`?